src/puget/ansi.clj

```diff
@@ -1,7 +1,8 @@
 (ns puget.ansi
```

Comment:
I think that this (and the `puget.html` namespace below) should be moved to `puget.color.ansi` and `puget.color.html`, respectively.